src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -908,18 +1099,13 @@ public void testStartIndividualArchive() throws OpenTokException {
                                 "          \"url\" : null,\n" +
                                 "          \"outputMode\" : \"individual\"\n" +
                                 "        }")));
-
         ArchiveProperties properties = new ArchiveProperties.Builder().outputMode(OutputMode.INDIVIDUAL).build();
-
         Archive archive = sdk.startArchive(sessionId, properties);
         assertNotNull(archive);
         assertEquals(sessionId, archive.getSessionId());
         assertNotNull(archive.getId());
         assertEquals(OutputMode.INDIVIDUAL, archive.getOutputMode());
-
         verify(postRequestedFor(urlMatching(archivePath)));
-                // TODO: find a way to match JSON without caring about spacing
```

Comment:
I don't think we should be removing these comments

Reply:
https://github.com/opentok/Opentok-Java-SDK/pull/150#discussion_r212050754